### PR TITLE
add virtual package conf-librsvg2

### PR DESCRIPTION
**Pull request by zoggy**

### CODE REVIEW

File: packages/conf-librsvg2/conf-librsvg2.0/opam [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+]
+x-ci-accept-failures: [
+  "oraclelinux-7"
+  "oraclelinux-8"
+]
+synopsis: "Virtual package relying on Librsvg2 system installation"
+description:
+  "This package can only install if librsvg-2.0-dev is installed on the system."
+flags: conf
+version: "0"
```
Comment (mseri) [inline packages/conf-librsvg2/conf-librsvg2.0/opam:29]:
```suggestion
```

File: packages/conf-librsvg2/conf-librsvg2.0/opam [hunk 2]
```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+maintainer: "https://github.com/ocaml/opam-repository/issues"
+authors: "The gtksourceview programmers"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+homepage: "https://wiki.gnome.org/Projects/LibRsvg"
+license: "LGPL-2.1-or-later"
+build: [["pkg-config" "--short-errors" "--print-errors" "librsvg-2.0"]]
```
Comment (mseri) [inline packages/conf-librsvg2/conf-librsvg2.0/opam:7]:
```suggestion
build: [["pkg-config" "--short-errors" "--print-errors" "librsvg >= 2.0"]]
```
Shouldn't it be checked with the comparison?

Comment (zoggy) [reply packages/conf-librsvg2/conf-librsvg2.0/opam:7]: No, pkg-config considers that the name of the package is librsvg-2.0. The -2.0 is not a version constraint, it's just part of the package name. 
```
$ pkg-config --short-errors --print-errors "librsvg >= 2.0"
No package 'librsvg' found
```

Comment (mseri) [reply packages/conf-librsvg2/conf-librsvg2.0/opam:7]: I see, thanks!

**Comment by mseri** [comment]
Then this looks good

**Pull request merged**